vcs/git/patch.go

```diff
@@ -19,10 +19,7 @@ func isFileBinaryGit(filename string) (bool, error) {
 		return false, errors.Trace(err)
 	}
 
-	out, err := gitCMD("-C", repoRoot, "diff", "--no-index", "--numstat", "/dev/null", filename)
-	if err != nil {
-	    return false, errors.Trace(err)
-	}
+	out, _ := gitCMD("-C", repoRoot, "diff", "--no-index", "--numstat", "/dev/null", filename)
```

Comment:
Add a comment explaining why we're ignoring the error, as this is usually a "no no".